test/correctness/simd_op_check.cpp

```diff
@@ -192,17 +193,17 @@ struct Test {
         // Define a vectorized Func that uses the pattern.
         Func f(name);
         f(x, y) = e;
-        f.bound(x, 0, W).vectorize(x, vector_width);
+        f.bound(x, PAD, W-PAD).vectorize(x, vector_width);
```

Comment:
That means you have a load or store to a ramp such as img(ramp{negative_constant, stride, extent})

https://github.com/halide/Halide/blob/master/src/CodeGen_LLVM.cpp#L1580